src/context.js

```diff
@@ -107,6 +161,31 @@ function EvolvContext(store) {
     return true;
   };
 
+  /**
+   * Persist context key to sessionStorage.
+   *
+   * @param {String} key The key to persist.
+   * @param {boolean} [sessionOnly] If true, the value will only be persisted to sessionStorage.
+   *   Default: true
+   */
+  this.persist = function(key, sessionOnly) {
```

Comment:
There is a TypeScript definitions file for this package. Can you also add `persist()` to the interface of `EvolvContext`?

https://github.com/evolv-ai/javascript-sdk/blob/8c54927e317848629ee88de35be3dcac694bd83d/src/types.d.ts#L154-L168